tools/ci_build/build.py

```diff
@@ -1126,8 +1126,8 @@ def run_training_python_frontend_e2e_tests(cwd):
     import torch
     ngpus = torch.cuda.device_count()
     if ngpus > 1:
-        log.debug('RUN: mpirun -n {} {} orttraining_run_glue.py'.format(ngpus, sys.executable))
-        run_subprocess(['mpirun', '-n', str(ngpus), sys.executable, 'orttraining_run_glue.py'], cwd=cwd)
+        log.debug('RUN: mpirun -n -x NCCL_DEBUG=INFO {} {} orttraining_run_glue.py'.format(ngpus, sys.executable))
+        run_subprocess(['mpirun', '-n', 'NCCL_DEBUG=INFO', str(ngpus), sys.executable, 'orttraining_run_glue.py'], cwd=cwd)
```

Comment:
>'mpirun', '-n', 'NCCL_DEBUG=INFO', str(ngpus), sys.executable, 'orttraining_run_glue.py' [](start = 24, length = 88)

['mpirun', '-n', str(ngpus), '-x', 'NCCL_DEBUG=INFO', sys.executable, 'orttraining_run_glue.py']. And log above needs to update too.